docker-compose.yaml

```diff
@@ -25,7 +25,7 @@ services:
       test: "exit 0"
     environment:
      - SINK_PATH=http://localhost:8098/fhir
-     - SOURCE_URL=https://localhost:8099/openmrs
```

Comment:
Suggestion: Change both `SOURCE_URL` and `SERVER_URL` to `OPENMRS_URL` everywhere, including in `streaming-atomfeed/Dockerfile`. I think that way it is more clear what this env. variable is and it is consistent everywhere.